crates/ruff_python_ast/src/source_code/indexer.rs

```diff
@@ -114,4 +134,73 @@ import os
         let indexer: Indexer = lxr.as_slice().into();
         assert_eq!(indexer.continuation_lines(), [9, 12]);
     }
+
+    #[test]
+    fn string_lines() {
+        let contents = r#""this is a string""#;
+        let lxr: Vec<LexResult> = lexer::lex(contents, Mode::Module).collect();
+        let indexer: Indexer = lxr.as_slice().into();
+        assert!(
+            indexer.string_lines()[0].location.row() == 1
+                && indexer.string_lines()[0].end_location.row() == 1
+                && indexer.string_lines().len() == 1
+        );
```

Comment:
Nit: You can use `assert_eq(indexer.string_lines(), &vec![Range::new(...)])` to do the comparision. This has the advantage that the test prints a nice diff if `string_lines` is empty or contains additional elements.